open-metadata-implementation/access-services/data-engine/data-engine-client/src/main/java/org/odpi/openmetadata/accessservices/dataengine/client/DataEngineImpl.java

```diff
@@ -33,7 +33,7 @@
  * The Data Engine Open Metadata Access Service (OMAS) provides an interface for data engine tools to create
  * processes with ports, schemas and relationships. See interface definition for more explanation.
  */
-public class DataEngineImpl extends OCFRESTClient implements DataEngineClient {
+public class DataEngineImpl extends OCFRESTClient implements DataEngineClient { //TODO: Suggest to rename this class to DataEngineRESTClient
```

Comment:
You can rename the class and remove the TODO, since it's already renamed in the documentation